new-docs/zh-CN/advanced/broker.md

```diff
@@ -24,4 +24,205 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Broker
\ No newline at end of file
+# Broker
+
+Broker 是 Doris 集群中一种可选进程，主要用于支持 Doris 读写远端存储上的文件和目录，如 HDFS、BOS 和 AFS 等。
+
+Broker 通过提供一个 RPC 服务端口来提供服务，是一个无状态的 Java 进程，负责为远端存储的读写操作封装一些类 POSIX 的文件操作，如 open，pread，pwrite 等等。除此之外，Broker 不记录任何其他信息，所以包括远端存储的连接信息、文件信息、权限信息等等，都需要通过参数在 RPC 调用中传递给 Broker 进程，才能使得 Broker 能够正确读写文件。
+
+Broker 仅作为一个数据通路，并不参与任何计算，因此仅需占用较少的内存。通常一个 Doris 系统中会部署一个或多个 Broker 进程。并且相同类型的 Broker 会组成一个组，并设定一个 **名称（Broker name）**。
+
+Broker 在 Doris 系统架构中的位置如下：
+
+```text
++----+   +----+
+| FE |   | BE |
++-^--+   +--^-+
+  |         |
+  |         |
++-v---------v-+
+|   Broker    |
++------^------+
+       |
+       |
++------v------+
+|HDFS/BOS/AFS |
++-------------+
+```
+
+本文档主要介绍 Broker 在访问不同远端存储时需要的参数，如连接信息、权限认证信息等等。
+
+## 支持的存储系统
+
+不同的 Broker 类型支持不同的存储系统。
+
+1. 社区版 HDFS
+   - 支持简单认证访问
+   - 支持通过 kerberos 认证访问
+   - 支持 HDFS HA 模式访问
+2. 百度 HDFS/AFS（开源版本不支持）
```

Comment:
Delete this information in Baidu community and directly introduce the version supported

Reply:
thanks ,already modified